src/share/util/mct_mod.F90

```diff
@@ -98,6 +98,7 @@ module mct_mod
    use m_GlobalSegMap       ,only: mct_gsMap_gstorage     => GlobalStorage
    use m_GlobalSegMap       ,only: mct_gsMap_ngseg        => ngseg
    use m_GlobalSegMap       ,only: mct_gsMap_nlseg        => nlseg
+   use m_GlobalSegMap       ,only: mct_gsMap_OP           => OrderedPoints
```

Comment:
The commit that made this change on the ACME side is:

commit 89940159774427904f816beca6e240fb35840dda
Author: Andy Salinger <agsalin@sandia.gov>
Date:   Fri Apr 14 11:57:23 2017 -0600

    Fixes to get tests to pass.
    
    scripts_regression_tests pass
    acme_developer has 1 remaining fail: HOMME_P24.f19_g16_rx1.A
    
    Changes:
    HOMME and cprnc CMakeFile fixes
    Undo mct_mod change that broke CLM build
    Remove unused <list>1<\list> in /clm/cime_config/config_component.xml
    Removed redundant MPASLI entries in env_run.xml
    Resolve conflict in update_acme_tests.py that was missed